### PR TITLE
{bio}[foss/2022a] CellBender v0.3.1

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3161.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/boegel/49aaf81911bc427d98f07d1cb7183bf5 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen3

**Comment by boegel** [comment]
failing test report is due to known issue with PyTorch on `jsc-zen3` (Rocky 9.x), see https://github.com/easybuilders/easybuild-easyconfigs/issues/19746, we shouldn't let that block this PR imho...

File: easybuild/easyconfigs/c/CellBender/CellBender-0.3.1-foss-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    ('jupyter_console', '6.6.3', {
+        'source_tmpl': SOURCE_PY3_WHL,
+        'checksums': ['309d33409fcc92ffdad25f0bcdf9a4a9daa61b6f341177570fdac03de5352485'],
+    }),
+    ('jupyter', '1.0.0', {
+        'checksums': ['d9dc4b3318f310e34c82951ea5d6683f67bed7def4b259fafbfe4f1beb1d8e5f'],
+    }),
+    ('cellbender', version, {
+        'source_urls': ['https://github.com/broadinstitute/CellBender/archive'],
+        'sources': [{'download_filename': 'e2fb597.tar.gz', 'filename': '%(name)s-%(version)s.tar.gz'}],
```
Comment (bedroge) [inline easybuild/easyconfigs/c/CellBender/CellBender-0.3.1-foss-2022a.eb:67]: See https://github.com/easybuilders/easybuild-easyconfigs/pull/19764#pullrequestreview-1856391167

Comment (boegel) [reply easybuild/easyconfigs/c/CellBender/CellBender-0.3.1-foss-2022a.eb:67]:
```suggestion
        'sources': [{'download_filename': '%s.tar.gz' % local_commit, 'filename': '%(name)s-%(version)s.tar.gz'}],
```

**Comment by bedroge** [comment]
Test report by @bedroge
**FAILED**
Build succeeded for 1 out of 2 (1 easyconfigs in total)
interactive2 - Linux Rocky Linux 8.9, x86_64, AMD EPYC-Milan Processor (zen2), Python 3.6.8
See https://gist.github.com/bedroge/ea4e3da3478b59ee9f9bd20adbb2711c for a full test report.

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
interactive2 - Linux Rocky Linux 8.9, x86_64, AMD EPYC-Milan Processor (zen2), Python 3.6.8
See https://gist.github.com/bedroge/42e4441f0bf055b35ed346d08a1257b5 for a full test report.

**Review by bedroge — APPROVED** [review]
Lgtm

**Comment by bedroge** [comment]
Going in, thanks @boegel!

**Pull request merged**